app/routes.py

```diff
@@ -0,0 +1,6 @@
+from app import app
+
+
+@app.route('/check/')
+def check():
+    return "Success"
```

Comment:
three thoughts about this: 
- does it make sense to define this in the template? shouldn't every service decide on how to define it's health (and if we change the name of the checker, shouldn't we go for sth like `/status` (w.o. trailing slash)
- shouldn't the response payload be a json? (sth like `{ "success" : true }`

Reply:
My idea was to have a simple route to be able to quickly test if the template work at all, this route can then be modified or removed when implementing the service.